### DISCUSSION TITLE
Adding analyst VM  to grid  fails unable to download .tar.gz from manager

### DISCUSSION THREAD

**Discussion by lerou114** [Security-Onion-Solutions/securityonion#7862] [General]
I have attempted to install the analyst VM into my grid and when I attempt to install the roles and add the VM into the grid I am getting this error:

<img width="1313" alt="missingpackage" src="https://user-images.githubusercontent.com/8277856/165349705-14739145-90be-4b0f-a695-8c22b9e86533.PNG">
:
I am also getting the same result when I try and add another vm that I had created for a IDH node and when attempting to add it to the grid. I get the same message.

**Comment by dougburks** [comment]
Looks like you're getting `Permission denied` when it tries to SSH to your manager. Are you able to manually SSH to your manager with the `soremote` account?
